src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -126,4 +132,11 @@ class EthService(blockchain: Blockchain, appStateStorage: AppStateStorage) {
       highestBlock = appStateStorage.getEstimatedHighestBlock()))
   }
 
+  def sendRawTransaction(req: SendRawTransactionRequest): Future[SendRawTransactionResponse] = {
+    import SignedTransactions._
+    val signedTransaction = rlp.decode[SignedTransaction](req.data.toArray[Byte])
```

Comment:
This change however depends on @rtkaczyk PR as he added possibility to return error from services